packages/xo-server-sdn-controller/src/index.js

```diff
@@ -207,6 +219,7 @@ class SDNController extends EventEmitter {
       networkDescription: { type: 'string' },
       encapsulation: { type: 'string' },
       pifId: { type: 'string' },
+      encrypted: { type: 'boolean' },
```

Comment:
All params that could have obvious default values should be optional:
- avoid breaking the API by introducing new required params
- makes things easier when using `xo-cli`